### DISCUSSION TITLE
[8 Giugno 2023] Cache

### DISCUSSION THREAD

**Discussion by FeddyLix17** [sapienzastudentsnetwork/architettura-degli-elaboratori#62] [Cache (M-Z)]
![image](https://github.com/sapienzastudentsnetwork/archelab2223/assets/66442720/462ecd57-4f32-42e5-bb13-bd059f0c9575)

**Comment by AglaiaNorza** [comment]
dovrebbe essere così
![image](https://github.com/sapienzastudentsnetwork/architettura-degli-elaboratori/assets/75022526/c5ecc729-ab33-47c1-b5fb-93de6d368914)

**Reply by matypist** [reply]
> dovrebbe

Confermo la proposta di soluzione di @AglaiaNorza, tutto chiaro e corretto :+1:
